x-pack/plugins/infra/server/routes/infra/lib/mapper.ts

```diff
@@ -20,10 +20,12 @@ import {
   HostsMetricsSearchValueRT,
 } from './types';
 import { METADATA_AGGREGATION_NAME } from './constants';
+import { HostAlertsResponse } from './host/get_hosts_alerts_count';
 
 export const mapToApiResponse = (
   params: GetInfraMetricsRequestBodyPayload,
-  buckets?: HostsMetricsSearchBucket[] | undefined
+  buckets?: HostsMetricsSearchBucket[] | undefined,
```

Comment:
Do we need the `| undefined` when it is optional?

Reply:
Yeah true, sorry, it looked like a new change in the diff but it's because of the `,` at the end.